### PR TITLE
Issue208

### PR DESCRIPTION
**Pull request by zorkow**
Rebuilds distribution to integrate new SRE as discussed in issue #208.

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ module.exports = function(grunt) {
       'speech-rule-engine': {
         options: {
           repository: 'https://github.com/zorkow/speech-rule-engine.git',
-          branch: 'v2.0.0-beta.2',
-          cwd: "."
+          branch: 'v2.0.0-beta.3',
```
Comment (pkra) [inline Gruntfile.js]: I think this should wait until v2 is out of beta.

Comment (zorkow) [inline Gruntfile.js]: A cheap shot.... 
But seriously here is my point:
I hated renaming JSON files to .js files. If it does not solve the problem I would revert that change before the 2.0 release. So I would like to test this first.
Given that there will be no more changes between beta.3 and 2.0 if it works, all we would need to do is update this line of code, without having to do another MathJax point release.

Would we really need another

Comment (pkra) [inline Gruntfile.js]: > Given that there will be no more changes between beta.3 and 2.0 if it works

Famous last words :wink: 

Ok with me then.

Comment (zorkow) [inline Gruntfile.js]: I've already written the release notes https://github.com/zorkow/speech-rule-engine/wiki/Release-Notes-v2.0
There might still be an update to the README, but that's it!

Comment (dpvc) [inline Gruntfile.js]: I'm OK with it as well.

**Comment by dpvc** [comment]
I think we can move on this.

**Pull request merged**